atlp-rwanda/ecommerce-app-legends-bn
# bug for correcting email template

### what this pr do:

To enable the app to send emails using the same template and format.

### how to test it manually:

- `git clone` from repo
- switch into branch `bg-email-template`
- run `npm install`
- run `npm run dev`
- with swagger documentation use this link `localhost:{port}/docs`
- login as a buyer and check your email

### screenshoots:
<img width="1440" alt="Screen Shot 2023-03-26 at 18 47 56" src="https://user-images.githubusercontent.com/99626447/227791055-b8a6905d-5239-46de-8f01-11ee1900cf54.png">
